lms/static/sass/_build-lms.scss

```diff
@@ -76,6 +75,10 @@
 // news
 @import 'news';
 
+@import 'mixins';
+@import 'mixins-inherited';
+@import 'elements/system-feedback';
```

Comment:
@marcotuts if we add these above in the start of file then we will get conflicts with a burbon mixin named as `button.`